src/Cake.Common.Tests/Fixtures/XmlPeekFixture.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+using System.Linq;
+using System.Xml;
+using Cake.Common.Tests.Properties;
+using Cake.Common.Xml;
+using Cake.Core;
+using Cake.Core.IO;
+using Cake.Testing;
+using NSubstitute;
+
+namespace Cake.Common.Tests.Fixtures
+{
+    internal sealed class XmlPeekFixture
```

Comment:
Addressed in [this commit](https://github.com/patridge/cake/commit/3c174c13e35ec06a0b0010c6e038f47fd6dd8b80).